src/api/app/controllers/webui/request_controller.rb

```diff
@@ -327,6 +323,11 @@ def chart_build_results
     render partial: 'webui/request/chart_build_results', locals: { chart_build_results_data: build_results_data }
   end
 
+  def complete_build_results
+    filters = params.keys.reject! { |key, _| key.in?(['controller', 'action', 'number']) }
```

Comment:
```suggestion
    filters = params.permit(:controller, :action, :number).keys
```
You can leverage [Strong Parameters](https://api.rubyonrails.org/classes/ActionController/StrongParameters.html) to do the same